CHANGES.rst

```diff
@@ -214,6 +214,9 @@ New Features
 
   - Added ``units.astronomical_unit`` as a long form for ``units.au``. [#3303]
 
+  - Clarified imperial mass measurements and added pound force (lbf),
+    kilopound (kip), and pound per square inch (psi)
```

Comment:
I think this will be going into v1.1.0, so eventually this will have to be moved before merging (and a reference to `[#3409]`) but good otherwise.  I need to add changelog sections for v1.1.0...